src/atomate2/common/flows/defect.py

```diff
@@ -348,8 +350,8 @@ def make(
         )
 
     @abstractmethod
-    def structure_from_prv(self, previous_dir: str) -> Structure:
-        """Copy the output structure from previous directory.
+    def sc_entry_and_locpot_from_prv(self, previous_dir: str) -> Structure:
```

Comment:
The returned object here should be `ComputedStructureEntry` and `Locpot` as docstring indicates, right? It is not longer `Structure`.

Reply:
correct